### PR TITLE
[camunda][SIMS#577] implementing fixes after new workflows

### PR DESCRIPTION
**Pull request by abschwenker**
1. Fix for studentPDStatus. Value was true/false and workflow was trying to validate "yes"/"no"
2. Fix for disbursement schedule not able to complete if 1 disbursement. Flow pointed to a decision node and should have bypassed it. Causing condition unable to evaluate

### CODE REVIEW

**Review by ann-aot — COMMENTED** [review]
👍

**Review by guru-aot — APPROVED** [review]
LGTM

**Review by ann-aot — APPROVED** [review]
👍

**Review by andrepestana-aot — APPROVED** [review]
LGTM

File: sources/packages/workflow/BPMN/camunda-8/assessment-2021-2022.bpmn [hunk 1]
```diff
@@ -1365,7 +1365,7 @@ or (calculatedDataPartnerFedralProvincialPDReceiptCost &gt; 0)</bpmn:conditionEx
     <bpmn:startEvent id="StartEvent_1" name="start">
       <bpmn:extensionElements>
         <zeebe:ioMapping>
-          <zeebe:output source="=institutionLocationProvince" target="institutionLocationProvince" />
+          <zeebe:output source="=if is defined(institutionLocationProvince) = false then null else institutionLocationProvince" target="institutionLocationProvince" />
```
Comment (andrewsignori-aot) [inline sources/packages/workflow/BPMN/camunda-8/assessment-2021-2022.bpmn:1368]: Minor but let's review this logic that is used in more than one place.
If this is to allow the IntelliSense when a variable is also possibly received, other points in the workflow declare the variables at the start event like below:
image

Comment (abschwenker) [reply sources/packages/workflow/BPMN/camunda-8/assessment-2021-2022.bpmn]: removed due to #1554

**Review by andrewsignori-aot — APPROVED** [review]
👍

**Pull request merged**